my-app/src/components/home page/menubar.js

```diff
@@ -18,94 +18,82 @@ import Events from './events';
 import FormsPage from '../forms/forms';
 import SportsPage from '../sports/sports';
 import Gallery from './carouselHome';
+import CalendarPage from '../calendar/calendar'
 
-/* TODO: add link to sports page on menu bar */
+/*TODO: add link to sports page on menu bar*/
 class Menu extends React.Component {
   render() {
-    return (
-      <Router>
-        <div className="menu">
-          <img src="/CalPolyLogo.png" alt="Cal Poly Logo" />
-          <h1>CLUB SPORTS HUB</h1>
-          <Link to="/login">LOGIN</Link>
-          <Link to="/forms">FORMS</Link>
-          <SportsDropDown />
-          <Link to="calendar">CALENDAR</Link>
-          <Link to="/">HOME</Link>
-        </div>
-        ;
-        <Switch>
-          <Route exact path="/" component={HomePage} />
-          <Route path="/forms" component={FormsPage} />
-          <Route exact path="/register" component={Register} />
-          <Route path="/login">
-            <Login />
-            ;
-          </Route>
-        </Switch>
-      </Router>
-    );
-  }
-}
-
-class SportsDropDown extends React.Component {
-  render() {
-    return (
-      <Router>
-        <div className="dropdown">
-          <button className="dropbtn">
-            SPORTS
-            <i className="fa fa-caret-down" />
+      return <Router>
+          <div className="menu">
+              <img src='/CalPolyLogo.png' alt='Cal Poly Logo' />
+              <h1>CLUB SPORTS HUB</h1>
+              <Link to="/login">LOGIN</Link>
+              <Link to="/forms">FORMS</Link>
+              <div class="dropdown">
+          <button class="dropbtn">SPORTS
+              <i class="fa fa-caret-down"></i>
           </button>
-          <div className="dropdown-content">
-            <div className="row">
-              <div className="column">
-                <h3>MEN&apos;S</h3>
-                <h4>__________</h4>
-                <b href="#">BASKETBALL</b>
-                <b href="#">LACROSSE</b>
-                <b href="#">HOCKEY</b>
-                <b href="#">RUGBY</b>
-                <b href="#">SOCCER</b>
-                <b href="#">ULTIMATE</b>
-                <b href="#">VOLLEYBALL</b>
-                <b href="#">WATER POLO</b>
+          <div class="dropdown-content">
+              <div class="row">
+                  <div class="column">
+                      <h3>MEN'S</h3>
+                      <h4>__________</h4>
+                      <Link to="/sports">BASKETBALL</Link>
```

Comment:
Are all of these links supposed to redirect to the same link? Doesn't quite make sense because from a user's standpoint there's no distinguishing between the different sport pages based on the URL.

Reply:
that is just temporary.  I am trying to find the best way to do that